changelog-generator/Cargo.toml

```diff
@@ -7,5 +7,7 @@ edition = "2021"
 
 [dependencies]
 serde = { version = "1.0.141", features = ["derive"] }
-serde_json = { version = "1.0.82", features = ["preserve_order"] }
+serde_json = { version = "1.0.83", features = ["preserve_order"] }
```

Comment:
if you're using toml, you don't need this anymore i presume

Reply:
I do as im still loading the responses from the API.  And they are json format, I did remove it(msitake) in the inertia of changing to toml.